region/brinstar/blue.json

```diff
@@ -11,7 +11,7 @@
           "id": 1,
           "name": "Far Left Door",
           "nodeType": "door",
-          "nodeSubType": "grey",
+          "nodeSubType": "blue",
```

Comment:
That brings up a question. Is grey a legit door color? I'm not sure which one this should be.

Reply:
Seems the colors do nothing anyway so *shrugs*